docs/book/operator-guide/auth-publisher.md

```diff
@@ -0,0 +1,126 @@
+be HS256.
+
+### JWT Expiration
+
+If the parameter is set and greater than 0, the publisher will check if the JWT token is expired
+based on the "issued at" (`iat`) value in the JWT token.
+
+### Upload parameters verification
+
+If set, the publisher will verify that the requested upload matches the claims in the JWT. This
+*does not* enable or disable the cryptographic authentication of the JWT; it just enables or
+disables the checks that ensure the contents of the JWT claim match the requested blob upload.
+
+Specifically, the publisher will:
+
+- Verify that the number of `epochs` in query is the the same as `epochs` in the JWT, if present;
+- Verify that the `send_object_to` field in the query is the same as the `send_object_to` in the
+  JWT, if present;
+- Verify the size of uploaded file;
+- Verify the uniqueness of the `jti` claim.
+
+### Replay-suppression configuration
+
+As mentioned above, the publisher supports replay suppression to avoid the malicious reuse of JWT
+tokens.
+
+The replay suppression supports the following configuration parameters:
+
+- `--jwt-cache-size`: The maximum size of the publisher's JWT cache, where the `jti` JWT IDs of the
+  "used" JWTs are kept until their expiration. This is a hard upperbound on the number of entries in
+  the cache, after which additional requests to store are rejected. This hard bound is introduced to
+  avoid DoS attacks on the publisher through the cache.
+- `--jwt-cache-refresh-interval`: The interval (in seconds) after which the cache is refreshed, and
+  expired JWTs are removed (possibly creating space for additional JWTs to be inserted).
+
+## JWT Fields
+
+The current authenticated publisher implementation does not provide a way to generate the JWTs and
+distribute them to the clients. These can be generated with any tool, as long as they respect the
+following constraints.
+
+### Mandatory fields
+
+- `exp` (Expiration): timestamp when the token expires;
+- `jti` (JWT ID): unique identifier for the token to prevent replay attacks;
+
+### Optional fields
+
+- `iat` (Issued At): Optional timestamp when the token was issued;
+- `send_object_to`: Optional Sui address where the newly-created `Blob` object should be sent;
+- `epochs`: Optional exact number of epochs the blob should be stored for
+- `max_epochs`: Optional maximum number of epochs the blob can be stored for
+- `max_size`: Optional maximum size of the blob in bytes
+- `size`: Optional exact size of the blob in bytes
+
+Note: The `epochs` and `max_epochs` claims cannot be used together, and neither can `size` and
+`max_size`. Using both in either case will result in token rejection.
+
+## Request authentication flow
+
+- The token is sent in the Authorization header as a Bearer token
+- The publisher verifies:
+  - Token signature using the configured secret
+  - Token expiration
+  - Token uniqueness (prevents replay attacks)
+  - Upload parameters against claims (if enabled)
```

Comment:
This doc is a good start but needs to describe the common case: how do I ensure as a publisher that an authenticated user can use up to X WAL/SUI for uploads. What combination of the above config should I use?